app/src/components/market/common/market_scale/value_box.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import ReactTooltip from 'react-tooltip'
+import styled from 'styled-components'
+
+import { IconInfo } from '../../../common/tooltip/img/IconInfo'
+import { Circle } from '../../common/common_styled'
+import { SCALE_HEIGHT, VALUE_BOXES_MARGIN } from '../common_styled'
+
+const ValueBoxSegment = styled.div`
+  padding: 12px;
+  border: 1px solid ${props => props.theme.scale.box};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 50%;
+  background: white;
+
+  &:nth-of-type(odd) {
+    border-top-right-radius: 0px;
+    border-bottom-right-radius: 0px;
+    border-top-left-radius: 4px;
+    border-bottom-left-radius: 4px;
+  }
+  &:nth-of-type(even) {
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+    border-top-left-radius: 0px;
+    border-bottom-left-radius: 0px;
+    border-left: none;
+  }
+`
+
+const ValueBoxSingle = styled.div<{ xValue?: number }>`
+  padding: 12px;
+  border: 1px solid ${props => props.theme.scale.box};
```

Comment:
also here i think we have borderLineDisabled in theme which has 1px solid with the desired color...